pages/includes/scripts.js

```diff
@@ -2,7 +2,6 @@ import '../../index.js';
 import 'uno.css';
 import { toast, updateToast, removeToast } from '../../index.js';
 import { WarpToastContainer } from '../../packages/toast/toast-container.js';
-import '@warp-ds/icons/elements';
```

Comment:
Are icons used in other components? If yes, we'll need to add individual imports for those or else those icons won't work.